src/training/params.py

```diff
@@ -66,6 +66,18 @@ def parse_args(args):
         default="title",
         help="For csv-like datasets, the name of the key for the captions."
     )
+    parser.add_argument(
+        "--hf-img-key",
+        type=str,
```

Comment:
What kind of hf dataset do you use ?

Reply:
also curious what hf datasets are being used? I'm not aware of any train datasets that would be fitting the FLAVA @ scale objective? Also HF Datasets needs some work before it's going to scale well to large multi-modal datasets for big cluster runs off shared storage. All of our datasets on clusters being used for training right now are in webdataset tars...

Reply:
I'm curious about this too! I tried HF dataset once for diffusion fine-tuning ([huggingface/diffusers ref](https://github.com/huggingface/diffusers/pull/356)), but it didn't really scale well + it was a bit hard to debug with Arrow format.

Reply:
@gmittal Can you reply here?